src/libraries/System.Net.Http/src/System/Net/Http/Headers/ViaHeaderValue.cs

```diff
@@ -3,19 +3,20 @@
 // See the LICENSE file in the project root for more information.
 
 using System.Diagnostics;
+using System.Diagnostics.CodeAnalysis;
 using System.IO;
 using System.Text;
 
 namespace System.Net.Http.Headers
 {
     public class ViaHeaderValue : ICloneable
     {
-        private string _protocolName;
-        private string _protocolVersion;
-        private string _receivedBy;
-        private string _comment;
+        private string? _protocolName;
+        private string _protocolVersion = null!;
```

Comment:
I presume this is because of the default constructor? Is it being used anywhere? If not, it could probably be removed.

Reply:
It is used once, and these fields set after that, i could remove the default constructor and make that ref to use other constructor but don't want to make such change unless that is ok for code owners. This is very common pattern for this project, there were several classes having default internal constructor which is used few times internally and setting required fields immediately, if we want to update those code we need to udpate all for consistency